app/src/services/course.services.ts

```diff
@@ -1,60 +1,71 @@
-import axios from "axios";
+import axios from 'axios'
 
 // Backend URL from enviroment
-const backend_route = import.meta.env.VITE_BACKEND_URL;
+const backendUrl = import.meta.env.VITE_BACKEND_URL
 
 // Interface for posting course content
 export interface CourseInterface {
-  title: string;
-  description: string;
+  title: string
+  category: string
+  level: string
+  time: number
+  description: string
 }
 
+/**
+ * IN ALL METHODS THE TOKEN HAS BEEN COMMENTED OUT, SINCE WE DON'T HAVE A TOKEN YET
```

Comment:
This should not be done. Mock it! Also the token could probably also be used a bit smarter by defining it as a header once rather than in each function